ui/app/src/components/LegacySiteDashboard/LegacyRecentActivityDashlet/RecentActivityDashlet.tsx

```diff
@@ -220,13 +220,13 @@ export function RecentActivityDashlet() {
 
   // region Item Updates Propagation
   useEffect(() => {
-    const events = [deleteContentEvent.type, workflowEvent.type, publishEvent.type];
+    const events = [deleteContentEvent.type, workflowEvent.type, publishEvent.type, contentEvent.type];
```

Comment:
@rart this event was added to the list because of an scenario where it is needed for the dashlet to be up to date. Please ping me for more details